src/test/java/com/nirmata/workflow/WorkflowAssertions.java

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
this allows you to make domain-specific chained assertions.

ex. instead of
`assertThat(thing.getPartners()).contains("howdy")`

`assertThat(thing).partnersContain("howdy")`